java/README.md

```diff
@@ -120,6 +198,6 @@ The following arguments are accepted:
 * `concurrentTasks`: Number of concurrent tasks
 * `clients`: one of: all|jedis|lettuce|glide
 * `clientCount`: Client count
-* `host`: redis server host url
-* `port`: redis server port number
-* `tls`: redis TLS configured
+* `host`: glide server host url
```

Comment:
revert - or change to ValKey.  But glide is the client that connects to Redis/Valkey